hazelcast-client-new/src/main/java/com/hazelcast/client/impl/DefaultClientExtension.java

```diff
@@ -93,4 +96,20 @@ public SocketInterceptor getSocketInterceptor() {
     public SocketChannelWrapperFactory getSocketChannelWrapperFactory() {
         return new DefaultSocketChannelWrapperFactory();
     }
+
+    /**
+     * Returns a proxy for the corresponding service.
+     *
+     * @param service service for the proxy to create.
+     * @return proxy for the service.
+     * @throws java.lang.IllegalArgumentException if type is not known
+     */
+    @Override
```

Comment:
Why do you copy paste the javadoc? The Javadoc tool automatically copied the javadoc of the method you implement/override. No need to copy paste it. Copy pasting leads to all kinds of problems, e.g. inconsistencies.